scripting/get5.sp

```diff
@@ -1650,11 +1679,6 @@ public Action Timer_PostKnife(Handle timer) {
   EnsureIndefiniteWarmup();
 }
 
-public Action StopDemo(Handle timer) {
```

Comment:
This method was not used anywhere.

Reply:
Couldn't we use this method though as a helper for #381?

Reply:
We already have a helper. I created a new file called `recording.sp` and put everything GOTV-related into that:

https://github.com/splewis/get5/pull/821/files#diff-bd010f45d65060a909e17e0429128077905e24c818c5d4d9a85cc4cc83702571R1-R24